.github/workflows/stale.yml

```diff
@@ -0,0 +1,23 @@
+name: 'Label inactive PRs'
+on:
+  schedule:
+    - cron: '30 1 * * *'
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v7
+        with:
+          operations-per-run: 30
+          stale-pr-message: 'Since this Pull Request has not been updated in 60 days, it has been marked "inactive." This does not mean that it will be closed, though it may be moved to a "Draft" state.  This helps maintainers prioritize their reviewing efforts. You can pick the PR back up anytime - please ping us if you need a review or guidance to move the PR forward!  If you do not plan on continuing the work, please let us know so that we can either find someone to take the PR over, or close it.'
+          stale-pr-label: 'inactive'
+          days-before-pr-stale: 60
+          days-before-pr-close: -1
+          stale-issue-message: 'This issue has been marked "inactive" because it has been 365 days since the last comment. If this issue is still present in recent Matplotlib releases, or the feature request is still wanted, please leave a comment and this label will be removed. If there are no updates in another 30 days, this issue will be automatically closed, but you are free to re-open or create a new issue if needed. We value issue reports, and this procedure is meant to help us resurface and prioritize issues that have not been addressed yet, not make them disappear.  Thanks for your help!'
+          stale-issue-label: 'inactive'
+          days-before-issue-stale: 365
+          days-before-issue-close: 30
+          ascending: true
+          exempt-issue-labels: keep
```

Comment:
added exempt labels (which could be expanded) so that if we have things we don't want cleaned up we can "keep"

Reply:
Maybe the "Orphaned PR" label should be exempt, as it seems redundant to check those.

Reply:
Sure.   Not sure if I have the syntax right for the labels with spaces in them...

Reply:
This format seems to work in Iris
https://github.com/SciTools/iris/blob/main/.github/workflows/stale.yml#L67